kpet/__init__.py

```diff
@@ -34,14 +32,13 @@ def exec_command(args, commands):
     except SystemExit:
         pass
     except:  # noqa: E731 don't use bare except pylint:disable=bad-option-value
-        logging.error('While executing command "%s"', args.command)
+        print('Error: While executing command "%s"',
+              args.command, file=sys.stderr)
```

Comment:
This won't work. Did you want to use `"".format()` here?

Reply:
What do you mean this is not C?

You're right, should be fixed now